packages/host/app/components/card-catalog-modal.gts

```diff
@@ -204,6 +220,72 @@ export default class CardCatalogModal extends Component<Signature> {
     );
   }
 
+  get realmsWithCards(): RealmCards[] {
+    let realmCards: RealmCards[] = [];
+
+    if (this.results.length) {
+      for (let instance of this.results) {
+        let realm = realmCards.find((r) => r.name === instance.realmInfo?.name);
+        if (realm) {
+          realm.cards.push(instance.card);
+        } else {
+          realm = {
+            url: instance.realmInfo.url,
+            name: instance.realmInfo.name,
+            iconURL: instance.realmInfo.iconURL
+              ? new URL(instance.realmInfo.iconURL, this.cardService.defaultURL)
+                  .href
+              : null,
+            cards: [instance.card],
+
+            displayedCards: [],
```

Comment:
I think this can be removed now that the "pagination" lives in the card list component